test/jdk/javax/swing/JComboBox/DisabledComboBoxFontTestAuto.java

```diff
@@ -113,30 +114,43 @@ private static void testMethod() throws IOException {
                 eColor2 = new Color(enabledImage2.getRGB(x, y));
                 dColor1 = new Color(disabledImage.getRGB(x + 1, y));
                 dColor2 = new Color(disabledImage2.getRGB(x, y));
+            } else if (lafName.equals("Windows")) {
+                // In Windows LAF, the ComboBox sizes are different and
+                // that results in pixel offset of 1px for width and 1px for height.
+                eColor1 = new Color(enabledImage.getRGB(x, y));
+                eColor2 = new Color(enabledImage2.getRGB(x + 1, y - 1));
+                dColor1 = new Color(disabledImage.getRGB(x, y));
+                dColor2 = new Color(disabledImage2.getRGB(x + 1, y - 1));
+            } else if (lafName.equals("GTK+")) {
+                // In GTK LAF, the ComboBox sizes are different and
```

Comment:
Is there any way to get these internal offsets programmatically and not use constants?

Reply:
Removed the offsets as after having unicode blank character there is no need of these.